nuxeo-runtime/nuxeo-runtime-stream/src/main/resources/OSGI-INF/codec-service.xml

```diff
@@ -27,6 +27,10 @@
     <codec name="avroJson" class="org.nuxeo.runtime.codec.AvroCodecFactory">
       <option name="encoding">json</option>
     </codec>
+    <!--<codec name="avroConfluent" class="org.nuxeo.runtime.codec.AvroCodecFactory">-->
+      <!--<option name="encoding">confluent</option>-->
+      <!--<option name="schemaRegistryUrls">http://localhost:8081</option>-->
```

Comment:
Forgot to remove ?

Reply:
well it can serve as an example on how to contribute the avroConfluent codec but yes I can drop it